vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/ModuleLinkManager.java

```diff
@@ -45,7 +51,7 @@ public class ModuleLinkManager extends LinkManager<IModuleLink,IModule,IModuleLi
     private ModuleLinkDisplayRepository moduleLinkDisplayRepo;
 
     @Override
-    public List<IModuleLinkDisplay> getLinkDisplays(String spaceId) {
+    public List <IModuleLinkDisplay> getLinkDisplays(String spaceId) {
```

Comment:
please fix your settings, there should be no blank here.

Reply:
fixed